client/tests/integration/smartcontracts/executor_with_admin/src/lib.rs

```diff
@@ -2,172 +2,33 @@
 //! If authority is not `admin@admin` then default validation is used as a backup.
 
 #![no_std]
-#![allow(missing_docs, clippy::missing_errors_doc)]
 
 #[cfg(not(test))]
 extern crate panic_halt;
 
-use iroha_executor::{
-    data_model::evaluate::{EvaluationError, ExpressionEvaluator},
-    parse,
-    prelude::*,
-    smart_contract,
-};
+use iroha_executor::{parse, prelude::*, smart_contract};
 use lol_alloc::{FreeListAllocator, LockedAllocator};
 
 #[global_allocator]
 static ALLOC: LockedAllocator<FreeListAllocator> = LockedAllocator::new(FreeListAllocator::new());
 
+#[derive(Constructor, Entrypoints, ExpressionEvaluator, Validate, Visit)]
+#[omit(visit(visit_instruction))]
 struct Executor {
     verdict: Result,
     block_height: u64,
     host: smart_contract::Host,
```

Comment:
I thought about hiding these fields in an `ExecutorBase` auto-generated struct as @DCNick3 suggested in a [comment](https://github.com/hyperledger/iroha/pull/3896#discussion_r1329718902), but felt that some external code might rely on the current `Executor` layout and it would be hard to enforce this invariant. I could still create an issue if you feel this feature is relevant.